src/Tools/AzureBlobStorage/AzureBlobStorage.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(ToolTargetFrameworks)</TargetFrameworks>
+    <RuntimeIdentifiers>linux-x64;linux-musl-x64;win-x64</RuntimeIdentifiers>
+    <PackAsToolShimRuntimeIdentifiers>linux-x64;linux-musl-x64;win-x64</PackAsToolShimRuntimeIdentifiers>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <RootNamespace>Microsoft.Diagnostics.Tools.Monitor.Egress.AzureBlob</RootNamespace>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>disable</Nullable>
+    <PublishToBlob>true</PublishToBlob>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\dotnet-monitor\Strings.Designer.cs" Link="ExternalRefs\Strings.Designer.cs" />
+    <Compile Include="..\dotnet-monitor\LoggingEventIds.cs" Link="ExternalRefs\LoggingEventIds.cs" />
+    <Compile Include="..\dotnet-monitor\LoggingExtensions.cs" Link="ExternalRefs\LoggingExtensions.cs" />
+    <Compile Include="..\dotnet-monitor\ExperienceSurvey.cs" Link="ExternalRefs\ExperienceSurvey.cs" />
```

Comment:
I think there are a lot of links that are not used. Are these brought in because of LoggingExtensions.cs?

Reply:
Most of this is due to `LoggingExtension`. I pulled that out and made a new one with just the few extensions that are needed and was able to remove this and pair down usage of all of this.